docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -262,6 +262,15 @@ $ kubectl --namespace teleport create -f aws-issuer.yaml
 ## Step 4b/7. Configure Teleport to use ACM to handle TLS
 
 <Admonition type="warning" title="Warning">
+  
+  Using ACM through a AWS Load Balancer prevents the required traffic for Postgres or MongoDB through the 
```

Comment:
Is there a more specific way to phrase "Using ACM through a AWS Load Balancer"? I'm not sure I understand this. Also, what does "required traffic" mean?